integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTestIso.java

```diff
@@ -117,7 +122,11 @@ public void lockBlockTest2() throws IOException {
     for (int k = 0; k < numOfFiles; k ++) {
       tFile = mTfs.getFile(fileIds.get(k));
       Assert.assertTrue(tFile.isInMemory());
-      Assert.assertNotNull(tFile.readByteBuffer(0));
+      is = tFile.getInStream(ReadType.CACHE);
+      buf = ByteBuffer.allocate((int) tFile.getBlockSizeByte());
+      is.read(buf.array());
+      Assert.assertNotNull(buf);
```

Comment:
I don't think this assert is doing the same thing as before. buf will always not be null if ByteBuffer.allocate() succeeds, regardless if the is.read() succeeds or fails.